inet.c

```diff
@@ -296,7 +299,7 @@ pcap_lookupdev(errbuf)
 		while(NAdapts--)
 		{
 			char* tmp = (char*)tUstr;
-			strcpy(tmp, tAstr);
+			strlcpy(tmp, tAstr, 8192);
```

Comment:
This code wasn't doing any bounds checking; I rewrote it to do bounds checking on the buffer, so it should now be OK without that change (it now uses `memcpy()` to do the copy - it's already determined the size of the string to copy, so that it can check if there's enough room in the buffer for the copy, so it can just copy all the bytes of the string, including the terminating NUL).